packages/components/src/components/my-position/my-position.scss

```diff
@@ -0,0 +1,66 @@
+@use "~@mapsindoors/midt/background-color";
+@use "~@mapsindoors/midt/border";
+
+@use "~@mapsindoors/midt/build/scss/variables";
+
+.mi-my-position {
+    margin: 10px;
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    pointer-events: auto;
+    gap: 16px;
+    right: 0;
+
+    &__position-button,
+        &__compass-button {
+            margin: 0;
+            cursor: pointer;
+            -webkit-appearance: none;
```

Comment:
I don't think you need the browser prefixes anymore.

https://caniuse.com/?search=appearance